src/js/heuristicblocking.js

```diff
@@ -130,6 +123,13 @@ HeuristicBlocker.prototype = {
 
     let tab_origin = self.tabOrigins[details.tabId];
 
+    // return early if tab origin is a localhost address
+    ['127.0', 'localhost', 'http://localhost'].forEach((localhost) => {
```

Comment:
Take a look at [`isPrivateDomain()`](https://github.com/EFForg/privacybadger/blob/a3bcf94e4b08581555c498c878945d2d0cd2e142/src/lib/basedomain.js#L152-L185).

Reply:
Oh nice! Forgot that was there, thanks